userspace/libsinsp/test/events_proc.ut.cpp

```diff
@@ -366,6 +367,29 @@ TEST_F(sinsp_with_test_input, spawn_process)
 	ASSERT_EQ(get_field_as_string(evt, "proc.ppid"), "1");
 	ASSERT_EQ(get_field_as_string(evt, "proc.apid[1]"), "1");
 	ASSERT_FALSE(field_exists(evt, "proc.apid[2]"));
+
+	// check more fields
+	ASSERT_EQ(get_field_as_string(evt, "proc.args"), "-c 'echo aGVsbG8K | base64 -d'");
+	ASSERT_EQ(get_field_as_string(evt, "proc.cmdline"), "test-exe -c 'echo aGVsbG8K | base64 -d'");
+	ASSERT_EQ(get_field_as_string(evt, "proc.exeline"), "/bin/test-exe -c 'echo aGVsbG8K | base64 -d'");
+	ASSERT_EQ(get_field_as_string(evt, "proc.tty"), "34818");
+	ASSERT_EQ(get_field_as_string(evt, "proc.vpgid"), "1");
+	ASSERT_EQ(get_field_as_string(evt, "user.loginuid"), "1000");
+	ASSERT_EQ(get_field_as_string(evt, "user.uid"), "2000");
+	ASSERT_EQ(get_field_as_string(evt, "proc.cwd"), "/root/");
+	ASSERT_EQ(get_field_as_string(evt, "proc.vmsize"), "29612");
+	ASSERT_EQ(get_field_as_string(evt, "proc.vmrss"), "4");
+	ASSERT_EQ(get_field_as_string(evt, "proc.vmswap"), "0");
+	ASSERT_EQ(get_field_as_string(evt, "proc.fdlimit"), "1024");
```

Comment:
looks like this is failing on ARM :thinking:

Reply:
that's weird, I don't see an obvious reason why this should fail

Reply:
Ok I think I already found a bug, working for a PR on that, but I did not get to the bottom of it yet. Well, there goes my afternoon :smiling_face_with_tear:

Reply:
@LucaGuerra no rush this PR is not urgent. Thanks for looking into it, will then rebase with the fix.

Reply:
It's a very fun bug, I think I'm close to the bottom of it but I'll have to resume on monday :sweat_smile:

Reply:
see PR comment